<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialSettingsWizard</class>
 <widget class="QWizard" name="InitialSettingsWizard">
  <property name="windowTitle">
   <string>Cantata First Run</string>
  </property>
  <widget class="WizardPage" name="introPage">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Cantata</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cantata is a client for Music Player Daemon (MPD). In order for Cantata to function you must already have MPD setup, configured, and running on either your local machine or a machine that is accessible to you.&lt;/p&gt;&lt;p&gt;For more information on MPD itself, please refer to the MPD website &lt;a href=&quot;http://www.musicpd.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://www.musicpd.org&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;br/&gt;&lt;p&gt;This 'wizard' will guide you through the basic settings required for Cantata to function correctly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>251</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="WizardPage" name="connectionPage">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connection details</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>The settings below are the basic settings required by Cantata. Please enter the relevant details, and use the 'Connect' button to test the connection.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="BuddyLabel" name="hostLabel">
      <property name="text">
       <string>Host (or local socket):</string>
      </property>
      <property name="buddy">
       <cstring>host</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="LineEdit" name="host"/>
    </item>
    <item row="5" column="0">
     <widget class="BuddyLabel" name="portLabel">
      <property name="text">
       <string>Port:</string>
      </property>
      <property name="buddy">
       <cstring>port</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="SpinBox" name="port">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>66000</number>
      </property>
      <property name="value">
       <number>6600</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="BuddyLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="buddy">
       <cstring>password</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="LineEdit" name="password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="BuddyLabel" name="dirLabel">
      <property name="text">
       <string>Music folder:</string>
      </property>
      <property name="buddy">
       <cstring>dir</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="PathRequester" name="dir"/>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="statusLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>165</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="musicFolderNoteLabel">
      <property name="text">
       <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; The 'Music folder' setting is used to lookup cover-art, lyrics, etc.&lt;/i&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLabel" name="socketNoteLabel">
      <property name="text">
       <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; When using a local socket the full absolute path to the socket needs to be set. (Port number is not required.)&lt;/i&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>19</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="WizardPage" name="filesPage">
   <layout class="QGridLayout" name="gridLayoutf">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6f">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Covers, Lyrics, and Streams</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_9f">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_5f">
      <property name="text">
       <string>&lt;p&gt;Cantata will download missing covers, and lyrics, from the internet. Cantata also allows you to save a list of internet streams (e.g. Radio Stations) that you wish to be able to play.&lt;/p&gt;&lt;p&gt;For each of these, please confirm whether you wish Cantata to store the relevant files within the music folder, or within your personal cache/config folders.&lt;/p&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_7x">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QFormLayout" name="filesFormLayout">
      <item row="0" column="0">
       <widget class="BuddyLabel" name="storeCoversInMpdDirLabel">
        <property name="text">
         <string>Save downloaded covers in music folder:</string>
        </property>
        <property name="buddy">
         <cstring>storeCoversInMpdDir</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="OnOffButton" name="storeCoversInMpdDir" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="BuddyLabel" name="storeLyricsInMpdDirLabel">
        <property name="text">
         <string>Save downloaded lyrics in music folder:</string>
        </property>
        <property name="buddy">
         <cstring>storeLyricsInMpdDir</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="OnOffButton" name="storeLyricsInMpdDir" native="true">
        <property name="text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="storeStreamsInMpdDirLabel">
        <property name="text">
         <string>Save list of streams in music folder:</string>
        </property>
        <property name="buddy">
         <cstring>storeStreamsInMpdDir</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="OnOffButton" name="storeStreamsInMpdDir" native="true">
        <property name="text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer_5x">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="persNote">
        <property name="text">
         <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; If you elect to have Cantata store covers, or lyrics, within the music folder, and you do not have write access to this folder, then Cantata will revert to saving the files in your personal cache folder. However, for streams, if you do not have write access, then you will only be able to view (and play) the streams stored in the music folder, and not add (or edit) any streams. If you elect not to store streams within the music folder, then these will be saved in your local config folder - and will only be accessible to yourself.&lt;/i&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_7f2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="httpNote">
      <property name="text">
       <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; The 'Music folder' is set to a HTTP address, and Cantata currently cannot upload files to external HTTP servers. Therefore, the above settings should be left disabled.&lt;/i&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <spacer name="verticalSpacer_8f">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="WizardPage" name="finishedPage">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Finished!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cantata is now connected to MPD! If you wish to further configure Cantata (to add extra MPD hosts, etc) then Canta's configuration dialog may be accessed from the menu triggered by the button in the top-right of Cantata's main window.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_6x">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>32</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="groupWarningIcon"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="groupWarningLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;b&gt;Warning:&lt;/b&gt; You are not currently a member of the 'users' group. Cantata will function better (saving of album covers, lyrics, etc. with the correct permissions) if you (or your administrator) add yourself to this group. If you do add yourself you will need to logout and back in for this to take effect.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WizardPage</class>
   <extends>QWizardPage</extends>
   <header>wizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathRequester</class>
   <extends>QLineEdit</extends>
   <header>pathrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OnOffButton</class>
   <extends>QWidget</extends>
   <header>onoffbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
